builtin/common/settings/dlg_settings.lua

```diff
@@ -514,7 +513,8 @@ local function get_formspec(dialogdata)
 		"box[0,0;", tostring(tabsize.width), ",", tostring(tabsize.height), ";#0000008C]",
 
 		("button[0,%f;%f,0.8;back;%s]"):format(
-				tabsize.height + 0.2, back_w, fgettext("Back")),
+				tabsize.height + 0.2, back_w,
+				fgettext(INIT == "pause_menu" and "Exit" or "Back")),
```

Comment:
this doesn't work and causes `pause_menu` to show up as a string in weblate

Reply:
```suggestion
				INIT == "pause_menu" and fgettext("Exit") or fgettext("Back")),
```

Reply:
https://github.com/luanti-org/luanti/commit/54bf5d62f2529d1444b109d685bb53401fca9d06